src/API/Rest/HealthStatusResponse.cs

```diff
@@ -0,0 +1,47 @@
+/*
+ * Apache License, Version 2.0
+ * Copyright 2019-2021 NVIDIA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+
+namespace Nvidia.Clara.DicomAdapter.API.Rest
+{
+    public enum ServiceStatus
+    {
+        Unknown,
+        Stopped,
+        Running,
+        Cancelled
+    }
+
+
+    /// <summary>
+    /// Response message of a successful inference request.
+    /// </summary>
+    public class HealthStatusResponse
```

Comment:
What is this based on? Is this indicating overall health status or active connection stats? Calling host only? or AE (allowing multiple from the same host)?

Reply:
This is DICOM Adapter internals.  Nothing to do with external devices.